Library/Homebrew/brew.sh

```diff
@@ -189,24 +189,30 @@ then
   if [[ -z "${HOMEBREW_BREW_WRAPPER:-}" ]]
   then
     odie <<EOS
-HOMEBREW_FORCE_BREW_WRAPPER was set to
-  ${HOMEBREW_FORCE_BREW_WRAPPER}
-but HOMEBREW_BREW_WRAPPER was unset. This indicates that you are running
-  ${HOMEBREW_BREW_FILE}
-directly but should instead run
-  ${HOMEBREW_FORCE_BREW_WRAPPER}
+conflicting Homebrew wrapper configuration!
+HOMEBREW_FORCE_BREW_WRAPPER was set to ${HOMEBREW_FORCE_BREW_WRAPPER}
+but HOMEBREW_BREW_WRAPPER was unset.
+
+Ensure you run $(bold "${HOMEBREW_FORCE_BREW_WRAPPER}") directly (not ${HOMEBREW_BREW_FILE})
+
+Manually setting your PATH can interfere with Homebrew wrappers.
+Ensure your shell configuration contains:
+  eval "\$(${HOMEBREW_BREW_FILE} shellenv)"
+or that $(bold "${HOMEBREW_FORCE_BREW_WRAPPER}") comes before ${HOMEBREW_BREW_FILE} in your PATH.
 EOS
   elif [[ "${HOMEBREW_FORCE_BREW_WRAPPER}" != "${HOMEBREW_BREW_WRAPPER}" ]]
   then
     odie <<EOS
-HOMEBREW_FORCE_BREW_WRAPPER was set to
-  ${HOMEBREW_FORCE_BREW_WRAPPER}
-but HOMEBREW_BREW_WRAPPER was set to
-  ${HOMEBREW_BREW_WRAPPER}
-This indicates that you are running
-  ${HOMEBREW_BREW_FILE}
-directly but should instead run:
-  ${HOMEBREW_FORCE_BREW_WRAPPER}
+conflicting Homebrew wrapper configuration!
+HOMEBREW_FORCE_BREW_WRAPPER was set to ${HOMEBREW_FORCE_BREW_WRAPPER}
+but HOMEBREW_BREW_WRAPPER   was set to ${HOMEBREW_BREW_WRAPPER}
+
+Ensure you run $(bold "${HOMEBREW_FORCE_BREW_WRAPPER}") directly (not ${HOMEBREW_BREW_FILE})
+
+Manually setting your PATH can interfere with Homebrew wrappers.
+Ensure your shell configuration contains:
+  eval "\$(${HOMEBREW_BREW_FILE} shellenv)"
+or that $(bold "${HOMEBREW_FORCE_BREW_WRAPPER}") comes before ${HOMEBREW_BREW_FILE} in your PATH.
```

Comment:
Should these be directories, instead of paths to executables? I worry a user may try to do something like
```
export PATH="/opt/homebrew/bin/brew:$PATH"
```
given this advice.

Reply:
@carlocab great point, will adjust and add an explicit `PATH` suggestion